pkgs/servers/forked-daapd/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv,
+  fetchFromGitHub,
+  avahi,
+  antlr3,
+  autoconf,
+  autoreconfHook,
+  curl,
+  ffmpeg,
+  gawk,
+  gettext,
+  gnutls,
+  gperf,
+  json_c,
+  minixml,
+  openssl,
+  libantlr3c,
+  libconfuse,
+  libevent,
+  libgcrypt,
+  libgpgerror,
+  libplist,
+  libsodium,
+  libunistring,
+  libuv,
+  libwebsockets,
+  pkg-config,
+  protobufc,
+  pulseaudio,
+  sqlite,
+  zlib,
+  spotifySupport ? false, libspotify ? null
+}:
+
+let
+  version = "27.2";
+  name = "forked-daapd";
+in stdenv.mkDerivation {
+  inherit name;
+
+  src = fetchFromGitHub {
+    repo = name;
```

Comment:
```suggestion
    repo = pname;
```

Reply:
I may be holding this wrong but these suggestions give me:

```
error: undefined variable 'pname' at /home/flo/nixpkgs/pkgs/servers/forked-daapd/default.nix:39:12
```